builtin/mainmenu/tab_settings.lua

```diff
@@ -190,22 +234,77 @@ local function parse_config_file()
 		return settings
 	end
 
-	-- store this helper variable in the table so it's easier to pass to parse_setting_line()
-	settings.current_comment = ""
+	parse_single_file(file, read_all, settings, 0)
 
-	local line = file:read("*line")
-	while line do
-		parse_setting_line(settings, line)
-		line = file:read("*line")
-	end
+	file:close()
 
-	settings.current_comment = nil
+	if parse_mods then
+		-- Parse games
+		local games_category_initialized = false
+		local index = 1
+		local game = gamemgr.get_game(index)
+		while game do
+			local file = io.open(game.path .. DIR_DELIM .. FILENAME, "r")
+			if file then
+				if not games_category_initialized then
+					table.insert(settings, {
+						name = "Games",
+						translation = fgettext_ne("Games"), -- not used, but needed for xgettext
```

Comment:
Perhaps make it more obvious that it really isn't needed, eg by making `translation` a local variable?